src/main/java/org/kiwiproject/dropwizard/util/lifecycle/ProcessIdLoggingServerLifecycleListener.java

```diff
@@ -10,6 +10,10 @@
 class ProcessIdLoggingServerLifecycleListener implements ServerLifecycleListener {
     private final Long processId;
 
+    ProcessIdLoggingServerLifecycleListener(long processId) {
```

Comment:
It would autobox it. This was just an absurdly slight optimization so it doesn't have to. And for all I know, the compiler maybe just does that automatically. But, I didn't think it would hurt anything.